apps/docs/pages/guides/getting-started/tutorials/with-nextjs.mdx

```diff
@@ -162,32 +323,58 @@ We can use the [Supabase Auth UI](/docs/guides/auth/auth-helpers/auth-ui) a pre-
 Install the Supabase Auth UI for React
 
 ```bash
-npm install @supabase/auth-ui-react
+npm install @supabase/auth-ui-react @supabase/auth-ui-shared
 ```
 
-Add the `Auth` component to your home page
+Create a `AuthForm` client side component with the `Auth` component imported in it
 
-```jsx title=pages/index.js
-import { Auth } from '@supabase/auth-ui-react'
-import { ThemeSupa } from '@supabase/auth-ui-shared'
-import { useSession, useSupabaseClient } from '@supabase/auth-helpers-react'
+```jsx title=app/authform.jsx
+// app/authform.jsx
+"use client";
+import { Auth } from "@supabase/auth-ui-react";
+import { ThemeSupa } from "@supabase/auth-ui-shared";
+import { useSupabase } from "./supabase-provider";
 
-const Home = () => {
-  const session = useSession()
-  const supabase = useSupabaseClient()
+export default function AuthForm() {
+  const { supabase } = useSupabase();
 
   return (
-    <div className="container" style={{ padding: '50px 0 100px 0' }}>
-      {!session ? (
-        <Auth supabaseClient={supabase} appearance={{ theme: ThemeSupa }} theme="dark" />
-      ) : (
-        <p>Account page will go here.</p>
-      )}
-    </div>
-  )
+    <Auth
+      supabaseClient={supabase}
+      view="magic_link"
+      appearance={{ theme: ThemeSupa }}
+      theme="dark"
+      showLinks={false}
+      providers={[]}
+      redirectTo="http://localhost:3000/auth/callback"
+    />
+  );
 }
+```
 
-export default Home
+Add the `AuthForm` component to your home page
+
+```jsx title=app/page.js
+// app/page.jsx
+import AuthForm from "./authform";
+
+export default async function Home() {
+  return (
+    <div className="row">
+      <div className="col-6">
+        <h1 className="header">Supabase Auth + Storage</h1>
+        <p className="">
+          Experience our Auth and Storage through a simple profile management
+          example. Create a user profile and upload an avatar image. Fast,
+          simple, secure.
+        </p>
```

Comment:
Confirming it makes sense to mention storage here?

Reply:
Yes it does.